docs/reference/overview_of_strategies.rst

```diff
@@ -1014,17 +1014,17 @@ following four-vectors :math:`(P(C \,|\, CC), P(C \,|\, CD), P(C \,|\, DC), P(C
 +------------------+-----------------------+--------------------------+
 | Name             | Four-vector           | Axelrod Library Name     |
 +==================+=======================+==========================+
-| `ZD-Extort-2`_   | (8/9, 1/2, 1/3, 0)    | :code:`ZDExtort2`        |
+| ZD-Extort-2      | (8/9, 1/2, 1/3, 0)    | :code:`ZDExtort2`        |
 +------------------+-----------------------+--------------------------+
-| `ZD-Extort-2v2`_ | (7/8, 7/16, 3/8, 0)   | :code:`ZDExtort2v2`      |
+| ZD-Extort-2v2    | (7/8, 7/16, 3/8, 0)   | :code:`ZDExtort2v2`      |
 +------------------+-----------------------+--------------------------+
-| `ZD-Extort-4`_   | (11/17, 0, 8/17, 0)   | :code:`ZDExtort4`        |
+| ZD-Extort-4      | (11/17, 0, 8/17, 0)   | :code:`ZDExtort4`        |
 +------------------+-----------------------+--------------------------+
-| `ZD-Gen-2`_      | (1, 9/16, 1/2, 1/8)   | :code:`ZDGen2`           |
+| ZD-Gen-2         | (1, 9/16, 1/2, 1/8)   | :code:`ZDGen2`           |
 +------------------+-----------------------+--------------------------+
-| `ZD-GTFT-2`_     | (1, 1/8, 1, 1/4)      | :code:`ZDGTFT2`          |
+| ZD-GTFT-2        | (1, 1/8, 1, 1/4)      | :code:`ZDGTFT2`          |
```

Comment:
On c0f0a4a you say there are code links but they don't actually link anywhere (should they?), you can see the table here (apologies for dragging you in to this, feel free to go silent at any point) http://axelrod.readthedocs.org/en/latest/reference/overview_of_strategies.html